component/libfuse/libfuse2_handler.go

```diff
@@ -209,10 +208,6 @@ func populateFuseArgs(opts *C.fuse_options_t, args *C.fuse_args_t) (*C.fuse_opti
 		options += ",allow_root"
 	}
 
-	if opts.non_empty {
```

Comment:
we need to pass this down to fuse also, why we are removing this here?

Reply:
Libfuse 3.0.0 release has removed the `nonempty` option. Please refer the PR description.